packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts

```diff
@@ -166,9 +164,9 @@ async function computeMembership(
   balances: OptionsWithBalances[],
 ): Promise<ComputedMembership> {
   const { requirements } = currentGroup;
-  const { isValid, messages } = await validateGroupMembership(
+  const { isValid, messages } = validateGroupMembership(
     address.address,
-    requirements,
+    requirements as Requirement[],
```

Comment:
How was this type error even making it through builds/CI?

Reply:
probably strict mode linting? not sure

Reply:
Why would strict mode linting stop a type error from being caught? Shouldn't we investigate further? Might point to an issue with our type-checking which we should fix immediately if it exists.

Reply:
Resolving to merge... we can continue this topic in the retro